### PR TITLE
API-Backend Test for Version Endpoint

### PR DESCRIPTION
**Pull request by adorapham**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):

Add API testing for Get version of FP
Also make the variable names for API Endpoint consistency across for back-end testing.

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

**Comment by adorapham** [comment]
Hi All, @Selbinyyaz and I work together on this PR. Please perform the review for us.

**Review by TimeaKovacs86 — DISMISSED** [review]
It run without problem for me. I guess it's alright.
Backend Test for Version Endpoint API passed

File: mocha/test/version.test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+describe("GET Version endpoint test", () => {
+  it("sending GET request with endpoint, status : 200 OK with response body", async () => {
+    let response = await apiHelper.sendGETRequest(APP_URL, apiEndPoint);
+    validator.validateResponse(response, {
+      statusCode: httpStatus.OK,
+    });
+
+    validator.validateResponse(response.body, {
+      commit: "unknown",
```
Comment (galeza) [inline mocha/test/version.test.js:17]: Hi @adorapham 
Maybe we could create an issue and ask why we have commit:"unknown". What is the purpose of this? Maybe "unknown" should be replaced by a value?

File: mocha/utils/apiEndpoints.js [hunk 2]
```diff
@@ -3,3 +3,4 @@ exports.feedbackEndpoint = "/api/feedback";
 exports.signUpApiEndpoint = "/api/auth/signup";
 exports.organisationsEndpoint = "/api/organisations";
 exports.postsEndpoint = "/api/posts";
+exports.versionApiEndpoint = "/api/version";
```
Comment (galeza) [inline mocha/utils/apiEndpoints.js]: @adorapham @jessica-lau 
what naming convention do you prefer? With or without APi? signUp**Api**Endpoint or signUpEndpoint?
Maybe we could unify that?

File: mocha/test/version.test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+const httpStatus = require("http-status");
+const APP_URL = process.env.MOCHA_URL;
+const apiEndPoints = require("../utils/apiEndpoints");
+const apiHelper = require("../utils/apiHelper");
+const validator = require("../utils/validators");
+
+let apiEndPoint = apiEndPoints.versionApiEndpoint;
+
+describe("GET Version endpoint test", () => {
+  it("sending GET request with endpoint, status : 200 OK with response body", async () => {
```
Comment (galeza) [inline mocha/test/version.test.js]: sending - with capital letter?

**Review by galeza — CHANGES_REQUESTED** [review]
LGTM

**Comment by galeza** [comment]
@adorapham 
Some minor comments. LGTM

File: mocha/test/version.test.js (hunk 3 above)
Comment (adorapham) [reply mocha/test/version.test.js]: Changed. TY

File: mocha/utils/apiEndpoints.js (hunk 2 above)
Comment (adorapham) [reply mocha/utils/apiEndpoints.js]: I prefer ApiEndpoint  as it seems a short of for the path -for example signUpApiEndpoint  /api/auth/signup, but I am fine with the other way .  After confirmation with Jessica, I will modify all to make sure we have the same format for all api endpoint. TY

File: mocha/test/version.test.js (hunk 1 above)
Comment (adorapham) [reply mocha/test/version.test.js:17]: @galeza , I create [PR#2121](https://github.com/FightPandemics/FightPandemics/issues/2121) like a question back to developers.   TY

File: mocha/test/version.test.js (hunk 3 above)
Comment (adorapham) [reply mocha/test/version.test.js]: Changed

File: mocha/utils/apiEndpoints.js (hunk 2 above)
Comment (jessica-lau) [reply mocha/utils/apiEndpoints.js]: I personally prefer without "Api" in the names as it keeps them simple and short.

Comment (adorapham) [reply mocha/utils/apiEndpoints.js]: Ok, so I will update all Endpoint without API and associated test.js files to this PR

Comment (adorapham) [reply mocha/utils/apiEndpoints.js]: @jessica-lau @galeza , I am done with the updated , please check

**Review by jessica-lau — DISMISSED** [review]
LGTM. thanks @adorapham for making the changes.

image

**Review by TimeaKovacs86 — DISMISSED** [review]
Looks good to me! TY 
API Backend Test for Version Endpoint passing

**Review by galeza — DISMISSED** [review]

**Review by JayashreeSamantasinghar — DISMISSED** [review]
All tests Passed. Looks good!! Thanks

**Comment by adorapham** [comment]
Thank you everyone.  hi @galeza -Please merge whenever you have time. TY

**Comment by galeza** [comment]
Hi @adorapham 
Please rebase or merge and remove conflicts: mocha/utils/apiEndpoints.js

**Comment by adorapham** [comment]
@galeza , I do the merged on staging with my updated on PR#2119 . Please check and merge. 

image

image

**Review by jessica-lau — APPROVED** [review]
LGTM
image

**Review by galeza — APPROVED** [review]

**Pull request merged**